runtime/v2/runc/task/service.go

```diff
@@ -548,7 +548,7 @@ func (s *service) checkProcesses(e runcC.Exit) {
 			}
 
 			p.SetExited(e.Status)
-			s.sendL(&eventstypes.TaskExit{
+			go s.sendL(&eventstypes.TaskExit{
```

Comment:
Since the intent here with `s.sendL` is to use the locking version of `s.send` in order to ensure event order, running this in a goroutine would be counterproductive.